src/framework/filterControl/FilterControl.jsx

```diff
@@ -183,7 +183,7 @@ export default class FilterControl extends Component {
 
 FilterControl.propTypes = {
   filterOptions: FilterOptionList.propTypes.filterOptions,
-  onAddFilter: PropTypes.func.isRequired,
+  onAddFilter: PropTypes.func,
   onRemoveSelectedFilter: FilterItem.propTypes.onRemoveSelectedFilter,
   onReplaceFilter: PropTypes.func.isRequired,
```

Comment:
Maybe `onReplaceFilter` should not be required as well? Since it is not needed in case of only one irremovable filter. 
`onAddFilter` is still required in the form classes.